<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChapterLinkDialog</class>
 <widget class="QDialog" name="ChapterLinkDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chapter Link</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../tensura.qrc">
    <normaloff>:/icons/assets/icons/bg-icon.png</normaloff>:/icons/assets/icons/bg-icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">border-radius: 6px;
background-color: rgb(20, 20, 20);</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="verticalSpacer_6">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>DroidSansMono NF</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">margin: 0 6px;
color: #ECECEC;
background-color: transparent;</string>
       </property>
       <property name="text">
        <string>Enter Chapter Link:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="linkInput">
       <property name="font">
        <font>
         <family>Iosevka NF</family>
        </font>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	padding: 6px;
	margin: 0 6px;
	color: #ECECEC;
	border: 0.5px solid #ECECEC;
	border-radius: 4px;
	background-color: transparent;
}</string>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Enter chapter link from one of the available sources...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="cancel_dialog_btn">
         <property name="font">
          <font>
           <family>DroidSansMono NF</family>
          </font>
         </property>
         <property name="toolTip">
          <string>Cancel</string>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius: 4px;
padding: 6px 18px;
background-color: rgb(244, 164, 173);
color: rgb(20, 20, 20);</string>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="load_chapter_btn">
         <property name="font">
          <font>
           <family>DroidSansMono NF</family>
          </font>
         </property>
         <property name="toolTip">
          <string>Load Link Chapter</string>
         </property>
         <property name="statusTip">
          <string>Load Link Chapter</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	border-radius: 4px;
	padding: 6px 18px;
	margin-right: 6px;
	background-color: rgba(85, 255, 255, 190);
	color: rgb(20, 20, 20);
}

QPushButton:hover {
	background-color: rgb(85, 255, 255);
}</string>
         </property>
         <property name="text">
          <string>Load</string>
         </property>
         <property name="icon">
          <iconset resource="../tensura.qrc">
           <normaloff>:/icons/assets/icons/load.jpeg</normaloff>:/icons/assets/icons/load.jpeg</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../tensura.qrc"/>
 </resources>
 <connections/>
</ui>
